week-2-practise/level-3/1-boolean.js

```diff
@@ -4,8 +4,8 @@
 // Leave comments above your changes to explain
 
 var isHappy = "false";
-
-if (isHappy == true) {
+//the double equal sign checks the value of isHappy. To check the type in addition to the value we us the triple equal
```

Comment:
That makes sense, but also one thing that this program is doing is that it's comparing a string to a boolean value in the if statement. Ideally, as a programmer you'd think twice before doing that - that's why the code is misleading. So you should be comparing a boolean to a boolean :)